packages/sitecore-jss/src/editing/graphql-editing-service.ts

```diff
@@ -132,23 +135,32 @@ export class GraphQLEditingService {
       language,
     });
 
+    /*
+    TODO: re-enable when dictionary schema updated
     dictionaryResults = editingData.site.siteInfo.dictionary.results;
     hasNext = editingData.site.siteInfo.dictionary.pageInfo.hasNext;
     after = editingData.site.siteInfo.dictionary.pageInfo.endCursor;
+    */
 
     while (hasNext) {
```

Comment:
I think you can just set _hasNext = false_ on line 128, so this loop will never be executed

Reply:
Dictionary query needs to be called at least once. It was separated from the first GQL service call to try-catch only the dictionary results.
This way we introduce minimal changes that will be the easiest to revert.

Reply:
but if you will set it to _false_ you don't need to use try/catch etc, so in order to revert back, you just need to set it back to _true_

Reply:
this code block will not be even executed: https://github.com/Sitecore/jss/blob/197ed42d21bc03daf32b96ef3926536c370f101c/packages/sitecore-jss/src/editing/graphql-editing-service.ts#L145